lib/parser.js

```diff
@@ -190,13 +190,23 @@ async function validateAndConvertMessage(msg, originalAsyncAPIDocument, fileForm
 
   const schemaFormat = msg.schemaFormat || DEFAULT_SCHEMA_FORMAT;
 
-  await PARSERS[String(schemaFormat)]({
+  const parser = PARSERS[String(schemaFormat)];
+
+  if (!parser) {
+    throw new ParserError({
+      type: 'schema-parser-not-registered',
+      title: 'The file does not correspond to a supported schema format',
+      detail: `The following message payload "${pathToPayload}" has "${schemaFormat}" schema 
```

Comment:
As @derberg suggest in first comment in another thread:

> we are missing validationErrors object that among other things contains exact jsonPointer that points to specific part of the document that is broken (containing schema format that is not supported by the parser) an also startLine and others
> 
> Have a look how this is done in custom validators https://github.com/asyncapi/parser-js/blob/master/lib/customValidators.js#L58

We should also add location object where error is occured. Please add it :)

Or we should add it in the followup PR? As I see, we probably don't have necessary data in the function to create this object. @derberg Any thoughts?